code/notificationbuilder/src/main/res/layout/push_template_product_rating_expanded.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/rating_expanded_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:theme="@style/DayNightTheme">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:clipToPadding="false"
+        android:clipChildren="false"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/large_icon"
+            android:layout_height="@dimen/large_icon_height"
+            android:layout_width="@dimen/large_icon_width"
+            android:layout_alignParentRight="true" />
```

Comment:
Good idea to center the image view to the center of the Layout 
```
android:layout_centerVertical="true"
```

This can make the notification to look better when notification title an notification body are multiline

Reply:
See https://github.com/adobe/aepsdk-ui-android/pull/11#discussion_r1617780965